### PR TITLE
commence work on package for shared resources

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis

### CODE REVIEW

**Comment by elharo** [comment]
#165

File: plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+
+  <extension point="org.eclipse.ui.keywords">
+    <keyword label="GCP" id="com.google.cloud.tools.eclipse.appengine.newproject.GCPKeyword"/>
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.xml]: - These labels should be put into the `plugin.properties` as they're internationalizable / searchable.
- The ids shouldn't have `new project`

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.xml]: done

**Comment by elharo** [comment]
PTAL @briandealwis

**Comment by briandealwis** [comment]
lgtm

**Pull request merged**